bigquery_etl/metadata/validate_metadata.py

```diff
@@ -192,6 +192,33 @@ def validate_exclusion_list_expiration_days(metadata, path):
 
     return is_valid
 
+def validate_retention_policy_based_on_table_type(metadata, path):
+    """Check if any of the retention exclusion tables have expiration_days set."""
+    is_valid = True
+    table_type = metadata.labels.get("table_type") if isinstance(metadata.labels, dict) else None
+    print(f"Table type: {table_type}")
+    print(metadata)
+    expiration_days = metadata.bigquery.time_partitioning.expiration_days
+    print(f"Expiration days: {expiration_days}")
```

Comment:
Is there a reason for these print statements? Or are they left over from testing this?

```suggestion
    expiration_days = metadata.bigquery.time_partitioning.expiration_days
```